src/include/templates/llm_min_prompt_template.hpp

```diff
@@ -0,0 +1,28 @@
+#ifndef LLM_MIN_PROMPT_TEMPLATE_H
+#define LLM_MIN_PROMPT_TEMPLATE_H
+
+constexpr auto llm_min_prompt_template = R"(
+You are RankLLM, an intelligent assistant that ranks tuples based on their relevance to the search query.
+
+I will provide you with some tuples, each indicated by a numerical identifier []. Identify and return the less relevant tuple to the search query.
```

Comment:
your min and max prompts should simply play with `return the { least | most } relevant tuple`.